### ISSUE TITLE
Package request: pipelight

### ISSUE THREAD

**Issue by thypon** [voidlinux/void-packages#1578]
http://pipelight.net/cms/about.html

Pipelight is a wrapper for using Windows plugins in Linux browsers and therefore giving you the possibility to access services which are otherwise not available for Linux users. Typical examples of such services are Netflix and Amazon Instant, which both use the proprietary browser plugin Silverlight. These services cannot normally be used on Linux since this plugin is only available for Windows, and the only open source alternative (Moonlight) is lacking support for DRM.

**Comment by beefcurtains** [comment]
Netflix is coming to Mozilla Firefox via integration with Adobe CDM: https://blog.mozilla.org/blog/2015/05/12/update-on-digital-rights-management-and-firefox/

Unity Web Player is still not supported on Linux, but you might find native versions of those games on Steam.

**Comment by thypon** [comment]
Sky Go isn't joining html5, i could say the same for a lot of local Netflix like services.

**Comment by beefcurtains** [comment]
Other alternatives to watch content on GNU+Linux are youtube-dl, livestreamer, and kodi.
In some cases, websites with Silverlight/Flash content have free counterparts in xbmc/kodi repositories.

**Comment by thypon** [comment]
I know them. I want to see DRMed content i payed for. Currently silverlight is the way they go. Until they replace it with another hipsy-gisty technology.

**Comment by beefcurtains** [comment]
Don't pay for restrictions, you're making the Stallman a very sad man. Install Popcorn Time :yum:

**Comment by thypon** [comment]
#1580

**Comment by JayVii** [comment]
Amazon Instant Video doesn't seem to work with adobe-flash-11.2 or pepperflash-17. Pipelight on my Debian-Machine gets it running though.
(through both adobe-flash-17 & silverlight-5.1)

"Install Popcorn Time" - I could get an insanely high fee for that in my country. P2P-streaming is no good :D

**Comment by beefcurtains** [comment]
How can you get a fee for installing Popcorn Time? It's perfectly legal to do.
Have you tried livestreamer to watch Amazon Instant Video?

**Comment by thypon** [comment]
@beefcurtains i know that in france they simple cut your line if you do P2P

**Comment by beefcurtains** [comment]
Can you still torrent a Linux iso?

**Comment by thypon** [comment]
Ahaha pretty fun. You download linux isos through Popcorntime? You stream them?

**Comment by beefcurtains** [comment]
Yes and [yes](https://github.com/mafintosh/torrent-mount), you can boot an iso while it's still downloading.

**Comment by JayVii** [comment]
You can get a fee for uploading copyright-protected data without permission & several court orders made clear, that this also applies to P2P & torrenting.
AFAIK (correct me, if i'm wrong), that's exactly what popcorntime does.

You cannot get harmed for streaming these movies/series/etc, but torrenting/P2P-ing movies is a completely different story.

**Comment by thypon** [comment]
@beefcurtains  Could you speak of torrenting on a social network? This is an issue opened for `pipelight` not a place to socialize or speak about butterflies.

**Comment by beefcurtains** [comment]
I removed myself from the discussion.

**Comment by thypon** [comment]
Right we would need or mingw32/64 or wine64 at least, unfortunatly neither of the two is available on voidlinux.
